routes/autopilot.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const twilio = require('twilio');
+const express = require('express');
+const assistant = require('../src/assistant');
+const Setup = require('../src/db').Setup;
+
+/* eslint-disable new-cap */
+const router = express.Router();
+
+// POST: /collect/main-menu
+router.post('/collect-main-menu', function(req, res, next) {
+  const options = JSON.parse(req.body.Memory).twilio.collected_data.options;
+  const answer = options.answers.option.answer;
+  if (options.status === 'complete') {
+    if (['sales', 'support'].includes(answer)) {
```

Comment:
I think a `switch` will be better than an `if..else`:
```javascript
switch(answer) {
  case 'sales':
  case 'support':
    ...
    break;
  case 'operator':
    ...
    break;
  default:
    res.send({
      ...
    });
}
```

Reply:
:100: